### PR TITLE
Success message with error toast

### LINKED ISSUE #1041: Success message with error toast
| Wazuh | Splunk | Rev |
| ----- | ------- | --- |
| 4.1.4-1   | 8.0.8     | 68|

**Description**
Error toast when with success message

**Steps to reproduce**

1. Go to management/groups
2. Edit agent.conf file
3. Save the new content
4. See the error

**Screenshots**

### PR DESCRIPTION
**Pull request by sortizowlh**
Hi team!
This PR removes invalid comparison with request data
Closes #1041

### CODE REVIEW

File: SplunkAppForWazuh/appserver/static/js/services/requestService/requestService.js [hunk 1]
```diff
@@ -171,6 +172,7 @@ define(['../module'], function(module) {
         }
         return result
       } catch (error) {
+        console.error(error);
```
Comment (Desvelao) [inline SplunkAppForWazuh/appserver/static/js/services/requestService/requestService.js:175]: Remove the console.log

File: SplunkAppForWazuh/appserver/static/js/services/requestService/requestService.js [hunk 2]
```diff
@@ -155,10 +155,11 @@ define(['../module'], function(module) {
           { content, origin: 'xmleditor' },
           'PUT'
         )
+
         if (
           !result ||
           !result.data ||
-          !result.data.data ||
+          // !result.data.data ||
```
Comment (frankeros) [inline SplunkAppForWazuh/appserver/static/js/services/requestService/requestService.js:162]: remove comment line

**Review by frankeros — CHANGES_REQUESTED** [review]

File: SplunkAppForWazuh/appserver/static/js/services/requestService/requestService.js (hunk 2 above)
Comment (sortizowlh) [reply SplunkAppForWazuh/appserver/static/js/services/requestService/requestService.js]: Comment removed!

File: SplunkAppForWazuh/appserver/static/js/services/requestService/requestService.js (hunk 1 above)
Comment (sortizowlh) [reply SplunkAppForWazuh/appserver/static/js/services/requestService/requestService.js]: Log removed!

**Comment by eze9252** [comment]
I tested and it works

**Review by eze9252 — APPROVED** [review]

**Pull request merged**